src/mozilla_sec_eia/models/sec10k/notebooks/exhibit21_extractor.ipynb

```diff
@@ -0,0 +1,1144 @@
+{
```

Comment:
<p>At a high level, I think I'm confused about whether this notebook should be used to run only validation, or also inference on the whole dataset. If it's only used for validation, then I think the job name and/or the name of the notebook should be changed to include "validation" in the name. It seems like this notebook is used for the&nbsp;<code>ex21_training</code>&nbsp;job and not&nbsp;<code>ex21_extraction</code>&nbsp;, but it's called&nbsp;<code>ex21_extraction.ipynb</code></p><p> </p><p>Also, maybe should be "This notebook implements a model built on top of&nbsp;<a href="https://huggingface.co/microsoft/layoutlmv3-base/tree/main" target="_blank">layoutlmv3</a>&nbsp;<strong>to extract tables</strong>&nbsp;from Exhibit 21 attachments to SEC-10k filings." I think the current wording doesn't make sense to me.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/mozilla-sec-eia/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>